python/pyspark/sql/functions.py

```diff
@@ -2681,6 +2681,30 @@ def overlay(src, replace, pos, len=-1):
     ))
 
 
+def sentences(str, lang="", country=""):
+    """
+    Splits a string into arrays of sentences, where each sentence is an array of words.
+    The 'lang' and 'country' arguments are optional, and if omitted, the default locale is used.
+
+    .. versionadded:: 3.2.0
+
```

Comment:
Can we also add:

```
    Parameters
    ----------
    str: ...
```

(see also https://numpydoc.readthedocs.io/en/latest/format.html)